azure-applicationinsights/azure/applicationinsights/operations/events_operations.py

```diff
@@ -120,7 +120,7 @@ def get_by_type(
 
         # Construct headers
         header_parameters = {}
-        header_parameters['Accept'] = 'application/json;odata=none'
+        header_parameters['Accept'] = 'application/json'
```

Comment:
why is that? Is this an Autorest issue? Something weird in the Swagger?

Reply:
Bug in the swagger on my end, see https://github.com/Azure/azure-rest-api-specs/pull/3409

Reply:
ok, saw your comment, thanks!